profiler/src/ProfilerEngine/Datadog.Profiler.Native/ManagedThreadList.cpp

```diff
@@ -270,6 +274,30 @@ HRESULT ManagedThreadList::TryGetCurrentThreadInfo(std::shared_ptr<ManagedThread
     }
 }
 
+bool ManagedThreadList::TryGetThreadInfo(uint32_t osThreadId, std::shared_ptr<ManagedThreadInfo>& ppThreadInfo)
+{
+    std::lock_guard<std::recursive_mutex> lock(_mutex);
+
+    if (_threads.empty())
+    {
+        return false;
+    }
+
+    auto elem = _lookupByOsThreadId.find(osThreadId);
+    if (elem == _lookupByOsThreadId.end())
+    {
+        return false;
+    }
+    else
+    {
+        ppThreadInfo = elem->second;
+        return true;
+    }
```

Comment:
```suggestion
    if (elem != _lookupByOsThreadId.end())
    {
        ppThreadInfo = elem->second;
        return true;
    }
```
maybe a little simpler

Reply:
✅  good idea